### ISSUE TITLE
Loss function problem

### ISSUE THREAD

**Issue by huytuong010101** [mjkwon2021/CAT-Net#11]
Hi @CauchyComplete  again 💯 
I am implementing your model with `tensorflow`. I try training it with CASIAv2/Tp (without pretrain) in 6 epoch but loss function reduce very low and not reduce since epoch 5. Can you share the plot of you loss without pretrain?
![loss](https://i0.wp.com/s1.uphinh.org/2021/11/15/Screenshot-2021-11-15-091530.png)

**Comment by huytuong010101** [comment]
Because it use SGD optimizer so do you think that the batch-size does not affect the loss?

**Comment by CauchyComplete** [comment]
I think the most critical problem is the lack of training data. I recommend you to collect more. Also, why don’t you add momentum in sgd?

**Comment by CauchyComplete** [comment]
@huytuong010101 
Here's the plot :grinning:
![log_extract](https://user-images.githubusercontent.com/24504012/143185968-04f8fc68-2e11-4342-9d15-c90d1eda51fb.png)

**Comment by huytuong010101** [comment]
> @huytuong010101 Here's the plot 😀 ![log_extract](https://user-images.githubusercontent.com/24504012/143185968-04f8fc68-2e11-4342-9d15-c90d1eda51fb.png)

Wow thank you so muchhhhhhhhh <3
